client/src/pages/certification.css

```diff
@@ -216,3 +216,14 @@
     margin-top: 20px;
   }
 }
+
+.certificate-outer-wrapper {
+  margin-top: calc(-1 * var(--header-height));
+  background-color: #fff;
+}
+
+.certificate-outer-wrapper .donation-completion-buttons button {
```

Comment:
@thecodingaviator, thanks for taking care of this. 
it seems like the buttons of donation completion are styled as default on learn.
To simplify the button styles and their hover styles,  I suggest using the same style for the other buttons.
I believe this #37938 should take care of all of them. So if we could take out the button styling, this pr should be good to go.